dm/ctl/master/operate_schema.go

```diff
@@ -27,7 +27,7 @@ import (
 // NewOperateSchemaCmd creates a OperateSchema command.
 func NewOperateSchemaCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-schema <operate-type> <-s source ...> <task-name> <-d database> <-t table> [schema-file]",
+		Use:   "operate-schema <operate-type> <-s source ...> <task-name | task-file> <-d database> <-t table> [schema-file]",
```

Comment:
Does `GetNameFromArgOrFile` need for `cmd.Flags().Arg(1)`?

Reply:
fix in https://github.com/pingcap/dm/pull/854/commits/7b5548113e45700a8e24536431d9c9aecac9fe83